packages/common/src/utils/test/certificateTestUtil.ts

```diff
@@ -204,7 +204,28 @@ export const getSickLeavePeriodElement = (): CertificateDataElement => {
     validationErrors: [],
   }
 }
-
+export const getDateRangeElement = (): CertificateDataElement => {
+  return {
+    id: '1.4',
+    parent: '1.1',
+    index: 1,
+    visible: true,
+    mandatory: false,
+    readOnly: false,
+    config: {
+      type: ConfigTypes.UE_DATE,
```

Comment:
Shouldn't ConfigType be UE_DATE_RANGE?

Reply:
Yes, I would prefer that aswell. I'll go ahead and add it!